e2e-polybft/README.md

```diff
@@ -0,0 +1,22 @@
+
+# End-to-End testing
+
+The implemented E2E tests start a local instance of V3.
+
+As such, they require the binary 'borv3' to be available in the $PATH variable.
```

Comment:
```suggestion
As such, they require the binary 'polygon-edge' to be available in the $PATH variable.
```

Anyhow, I'm not sure anymore how the client is called, PoS v3, Edge v3...? :smile: